modules/qtwidgets/include/modules/qtwidgets/ordinaleditorwidget.h

```diff
@@ -50,17 +50,27 @@ class IVW_MODULE_QTWIDGETS_API BaseOrdinalEditorWidget : public QWidget {
     Q_OBJECT
 #include <warn/pop>
 public:
-    BaseOrdinalEditorWidget();
+    BaseOrdinalEditorWidget(bool intMode);
     virtual ~BaseOrdinalEditorWidget();
 
 protected:
-    virtual QString transformValueToEditor() = 0;
-    virtual void newEditorValue(QString) = 0;
+    virtual double transformValueToEditor() = 0;
+    virtual double transformMinValueToEditor() = 0;
+    virtual double transformMaxValueToEditor() = 0;
+    virtual double transformIncrementToEditor() = 0;
+    virtual int transformIncrementToEditorDecimals() = 0;
+
+    virtual void newEditorValue(double) = 0;
 
     void applyInit();
     void applyValue();
+    void applyMinValue();
+    void applyMaxValue();
+    void applyIncrement();
 
-    IvwLineEdit* editor_;
+    NumberLineEdit* editor_;
+    ConstraintBehaviour minCb_;
```

Comment:
Picky, but this should be minCB_ I believe. Change if you want to

Reply:
agree